src/Mod/Path/PathScripts/PostUtilsParse.py

```diff
@@ -365,12 +379,23 @@ def parse(values, pathobj):
                                 )
                         else:
                             continue
-                    elif param in ["H", "L", "T"]:
+                    elif param in ("H", "L", "T"):
```

Comment:
Using tuples instead of lists because of immutability?   Fine by me.

Reply:
I prefer to use tuples instead of lists to imply that the data structure isn't going to be changed, where possible.
Since I am refactoring already existing code I am making changes like that when I spot the opportunity.  It wouldn't surprise me if there are other such opportunities that I haven't noticed so far.